### PR TITLE
virts-1291

### PR DESCRIPTION
**Pull request by ArtificialErmine**
Adds a remote.host.ip parser to arp retrieval so that the addresses can be used by the planner.

### CODE REVIEW

File: app/parsers/ipaddr.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                    IPs.append(Relationship(source=Fact(trait=mp.source, value=ip),
+                                            edge=mp.edge,
+                                            target=Fact(mp.target, '')))
+        return IPs
+
+    @staticmethod
+    def _is_valid_ip(raw_ip):
+        try:
+            # The following hardcoded addresses are not used to bind to an interface.
+            if raw_ip in ['0.0.0.0', '127.0.0.1']:  # nosec
```
Comment (elegantmoose) [inline app/parsers/ipaddr.py:25]: Define hard coded static IP exception list at top of module.

File: app/parsers/ipaddr.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                return False
+            if any([True if raw_ip.endswith(x) else None for x in ['.255', '.0', '.1']]):
```
Comment (elegantmoose) [inline app/parsers/ipaddr.py:27]: same here, define last-octet-exception list at top level of module

File: app/parsers/ipaddr.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                return False
+            ip_address(raw_ip)
+        except BaseException:
```
Comment (elegantmoose) [inline app/parsers/ipaddr.py:30]: Is there a reason we are catching BaseException here instead of Exception?

File: app/parsers/ipaddr.py (hunk 1 above)
Comment (ArtificialErmine) [reply app/parsers/ipaddr.py]: Yeah, makes sense.

Comment (ArtificialErmine) [reply app/parsers/ipaddr.py]: Yeah, makes sense.

**Pull request merged**